mandatory/4-tax.js

```diff
@@ -17,7 +20,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let taxOf = (price * 20) / 100;
```

Comment:
Consider variable name taxOf : a proper naming way will be more clearer when defnining variables.